### DISCUSSION TITLE
I can't find osc.exe.

### DISCUSSION THREAD

**Discussion by iverson1github** [ComputationalBiomechanicsLab/opensim-creator#1054] [Q&A] [answered]
After Run the build script build_windows.bat, I don't find the osc.exe in the osc-build directory. Why?

**Comment by adamkewley** [comment, marked as answer]
`build_windows.bat`, once sucesfully completed, builds the `package` target for `osc` (i.e. the installer):

- https://github.com/ComputationalBiomechanicsLab/opensim-creator/blob/main/scripts/build_windows.bat#L91

The `package` target also builds the `osc` target. The location of `osc.exe` is build-system-determined, but on Windows using MSVC etc. it will typically be something like `.\osc-build\x64-RelWithDebInfo\apps\osc\osc.exe`, or similar - it's usually easiest to just search for `osc.exe` in the build dir.

If you intend on developing `osc` on Windows with (e.g.) Visual Studio, there's a rough guide here:

- https://github.com/ComputationalBiomechanicsLab/opensim-creator#visual-studio-2022

Which initially builds all of OSC's dependencies, followed by hooking up VS with the build so that developers can edit, build, run, etc. OSC
